lib/Doctrine/ORM/EntityRepository.php

```diff
@@ -129,9 +131,13 @@ public function createNativeNamedQuery($queryName)
      * Clears the repository, causing all managed entities to become detached.
      *
      * @return void
+     *
+     * @deprecated 2.7 This method is being removed from the ORM and won't have any replacement
```

Comment:
This should be 2.8 since we don't deprecate things in patch releases.

Reply:
For this reason I'm going to change the destination branch here, please rebase your work against the branch 2.8.x 👍

Reply:
atm 2.8 and 2.7 is the same, so i dont know why to rebase.

Reply:
This is a minor contributions, and 2.7 only accepts patches.

Reply:
@greg0ire yes, but still the branches are same, wo why to git rebase?

Reply:
Oh right sorry, if it's the same there's no point indeed